src/fakeData/healthcareProfessionals.ts

```diff
@@ -0,0 +1,80 @@
+import { HealthcareProfessional, Locale, LocaleName, Degree, 
+    Specialty, SpecialtyName, SpokenLanguage, Insurance } from '../typeDefs/gqlTypes'
+
+export const fakeHealthcareProfessionals = () => {
+    const doctorDoctor : LocaleName = {
+        lastName: 'Doctor',
+        firstName: 'Doctor',
+        middleName: 'MD',
+        locale: Locale.English
+    }
+    const medicalDegree : Degree = {
+        nameJa: 'メヂカル',
+        nameEn: 'Medical',
+        abbreviation: 'MD'
+    }
+
+    const japanese : SpokenLanguage = {
+        iso639_3: 'ja',
+        nameJa: '日本語',
+        nameEn: 'Japanese',
+        nameNative: 'Japanese'
+    }
+
+    const neurologyEn : SpecialtyName = {
+        name: 'Neurology',
+        locale: Locale.English
+    }
+
+    const neurology : Specialty = {
+        names: [neurologyEn]
+    }
+
+    const healthcareProfessionalOne : HealthcareProfessional = {
+        names: [doctorDoctor],
+        degrees: [medicalDegree],
+        spokenLanguages: [japanese],
+        specialties: [neurology],
+        acceptedInsurance: [Insurance.InternationalHealthInsurance],
+        isDeleted: false
+    }
+
+    const name : LocaleName = {
+        lastName: 'チェ',
+        firstName: 'ジェイコブ',
+        middleName: 'ベイヤード',
+        locale: Locale.Japanese
+    }
+    const englishDegree : Degree = {
+        nameJa: '英語',
+        nameEn: 'English',
+        abbreviation: 'En'
+    }
+
+    const english : SpokenLanguage = {
+        iso639_3: 'en-US',
+        nameJa: '英語',
+        nameEn: 'English',
+        nameNative: 'English'
+    }
+
+    const specialtyName : SpecialtyName = {
+        name: 'Pandas',
+        locale: Locale.English
+    }
+
+    const pandaSpecialty : Specialty = {
+        names: [specialtyName]
+    }
+
+    const healthcareProfessionalTwo : HealthcareProfessional = {
+        names: [name],
+        degrees: [englishDegree],
+        spokenLanguages: [english],
+        specialties: [pandaSpecialty],
+        acceptedInsurance: [Insurance.InternationalHealthInsurance],
+        isDeleted: false
+    }
+
+    return [healthcareProfessionalOne, healthcareProfessionalTwo]
+}
```

Comment:
This file was also removed in #227

Reply:
@ermish I asked this in the issue, but I'll ask it again here. What can this fake data do that auto-mocking from the schema can't? It seems like extra work to manually seed the db with files that will have to be updated and maintained.

Reply:
answered in the previous comment:

> yes, we want fake data.
> this will be used for when we want to test endpoints by adding data, updating it, then deleting it.
> we will also want seed data for when we load up a fake/test database so the site isn't empty.
> Keep in mind this is just a starting point and we'll need to update this iteratively.